packages/server/dataloader/customLoaderMakers.ts

```diff
@@ -85,7 +85,11 @@ export const serializeUserTasksKey = (key: UserTasksKey) => {
   return parts.join(':')
 }
 
-export const commentCountByDiscussionId = (parent: RootDataLoader) => {
+export const commentCountByDiscussionId = (
+  parent: RootDataLoader,
+  dependsOn: RegisterDependsOn
+) => {
+  dependsOn('comments')
```

Comment:
custom dataloaders can register with a primary key loader.
when a mutation calls `dataLoader.clearAll('comments')` this dataloader, if initialized, will get wiped, too.